### PR TITLE
Refs #10782 - document Foreman statuses

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: _includes/manuals/1.10/4.9_monitoring.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To monitor your infrastructure, host statuses are useful. In Foreman each host
+has a *global* status that tells which hosts need attention. Each host also has
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9_monitoring.md]: s/tells/indicates/

File: _includes/manuals/1.10/4.9_monitoring.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+sub-statuses that represents status of a particular feature. With every change
+of sub-status a global status is recalculated and the result is determined by
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9_monitoring.md]: "With any change of a sub-status, the global status is [..]", or "On any sub-status change, the global status is [..]"

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+The global status represents the overall status of a particular host. The
+status can have one of three possible values - **OK**, **Warning**, **Error**.
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: OK, Warning or Error.

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+OK means that all sub-statuses do not report anything wrong. It is represented
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: This section could do with being in separate paragraphs or bullet points for each status.

"OK means that no errors were reported by any sub-status"?

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+with green color. Warning suggests that user should verify the status, while
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: "with the color green"

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+no error was detected, some sub-status raised a warning. A good example would
+be that there are no puppet reports for the host even though the host is
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: capitalise Puppet

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+configured to send puppet reports. Therefore it is highlighted with yellow
+color. Last possible value is Error which indicates that some sub-status
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: "with the color yellow"
"The last possible value is Error"

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+reports a failure. This could for example mean that puppet run contains
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: ditto, Puppet

File: _includes/manuals/1.10/4.9.1_global_status.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+some failed resources. Obviously it is something that should be fixed and
+is user is alerted by red color.
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: "the color red"

File: _includes/manuals/1.10/4.9.2_substatuses.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
+A sub-status monitors only a part of host capabilities. Currently Foreman ships
+only two - **build** and **configuration** sub-statuses. Not all sub-statuses
+are relevant for all hosts, therefore configuration is only considered if host
+is using some configuration management system e.g. has some puppet proxy
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.2_substatuses.md]: Puppet, and a comma before e.g.

File: _includes/manuals/1.10/4.9.2_substatuses.md [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+associated. Build sub-status is relevant for managed hosts and when Foreman is
+run in unattended mode.
+
+You can see a global host status with all sub-statuses on host detail page in
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.2_substatuses.md]: "on the host detail page, in the properties table."

File: _includes/manuals/1.10/4.9.2_substatuses.md [hunk 12]
```diff
@@ -0,0 +1,57 @@
...
+properties table. Note that there can be more sub-statuses added from plugins.
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.2_substatuses.md]: added by plugins

File: _includes/manuals/1.10/4.9.2_substatuses.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+
+![All statuses](/static/images/screenshots/4.9.2_all_statuses.png)
+
+Each sub-status can define own set of possible values that are mapped to three
+global status values. Build sub-status has two possible values - **pending**
+and **built** that are both mapped to global OK value. Configuration status
+is more complicated and possible values and mapping are described in table
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.2_substatuses.md]: "and its possible values and mappings"

File: _includes/manuals/1.10/4.9.2_substatuses.md [hunk 14]
```diff
@@ -0,0 +1,57 @@
...
+    <tbody>
+        <tr>
+          <th>Status value</th>
+          <th>Maps to</th>
+          <th>Description</th>
+        </tr>
+        <tr>
+          <td><strong><em>Error</em></strong></td>
+          <td>Error</td>
+          <td>Error during configuration, e.g. puppet run failed to install some package</td>
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.2_substatuses.md]: Puppet throughout this table.

File: _includes/manuals/1.10/4.9.2_substatuses.md [hunk 15]
```diff
@@ -0,0 +1,57 @@
...
+        </tr>
+        <tr>
+          <td><strong><em>Out&nbsp;of&nbsp;sync</em></strong></td>
+          <td>Warning</td>
+          <td>Configuration report was not received even though it was expected based on <em>puppet_interval</em> and <em>outofsync_interval</em> settings.</td>
+        </tr>
+        <tr>
+          <td><strong><em>No&nbsp;reports</em></strong></td>
+          <td>Warning&nbsp;/&nbsp;OK</td>
+          <td>When there are no reports but the host uses configuration management system (e.g. puppet proxy is associated) or <em>always_show_configuration_status</em> settings is set to true, it maps to Warning. Otherwise it is mapped to OK.</td>
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.2_substatuses.md]: "setting" singular

File: _includes/manuals/1.10/4.9.3_searching_by_statuses.md [hunk 16]
```diff
@@ -0,0 +1,24 @@
...
+
+* find hosts that have at least one pending resource
+
+    `status.pending > 0`
+
+* find hosts that restarted some service during last puppet run
+
+    `status.restarted > 0`
+
+* find hosts that have last puppet run interesting (something happened)
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.3_searching_by_statuses.md]: "have an interesting last Puppet run"

File: _includes/manuals/1.10/4.9.3_searching_by_statuses.md [hunk 17]
```diff
@@ -0,0 +1,24 @@
+You can search hosts by global status. Some examples can be found below
```
Comment (domcleal) [inline _includes/manuals/1.10/4.9.3_searching_by_statuses.md]: add a colon to the end of the line, ditto on 12

File: _includes/manuals/1.10/4.9.1_global_status.md (hunk 3 above)
Comment (ares) [inline _includes/manuals/1.10/4.9.1_global_status.md]: you meant to also remove the formatting or just replace last comma with or?

Comment (domcleal) [inline _includes/manuals/1.10/4.9.1_global_status.md]: Just the comma.

**Comment by ares** [comment]
@domcleal thanks for the review, I fixed all your comments.

**Comment by domcleal** [comment]
Thanks @ares!

**Pull request merged**